Change CONTENT_ORIGIN to REACT_APP_CONTENT_ORIGIN; drop GITHUB_ORG

sideshowbarker opened the pull request:
This change also removes the dotenv-webpack dependency.

sideshowbarker committed:
Change CONTENT_ORIGIN to REACT_APP_CONTENT_ORIGIN; drop GITHUB_ORG

This change also removes the dotenv-webpack dependency.

pommicket commented:
Looks good, but we'll need to make this change in `libs/constants/index.js`:
```diff
-export const DEFAULT_LOCALE = process.env.DEFAULT_LOCALE || "en-US";
+export const DEFAULT_LOCALE = process.env.REACT_APP_DEFAULT_LOCALE || "en-US";
```
so that the hack of setting the default locale to `en-us` still works.

sideshowbarker committed:
Use REACT_APP_DEFAULT_LOCALE

This change also includes some whitespace reformatting that gets
automatically made by the commit linter.
